ecosystem/sep-0008.md

```diff
@@ -159,8 +159,8 @@ In the future, this requirement can be replaced by introducing protocol level [C
 Ideally, no. Implementing Regulated Assets should only be used when absolutely necessary, such as for regulatory reasons. It comes with a substantial operational overhead, added complexity, and burden for users. Issuers should only go down this route if it is absolutely required.
 Alternatively, some other available options are to utilize [SEP006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md) in order to perform KYC on deposit and withdraw, and/or use [AUTHORIZATION_REQUIRED](https://www.stellar.org/developers/guides/issuing-assets.html#requiring-or-revoking-authorization) which allows issuers to authorize specific accounts to hold their issued assets.
 
-### Why doesn’t the approval service submit transactions to the network?
+### Why does the approval service submit transactions to the network?
 
-- Separation of concerns between signing transactions and submitting them.
-- Transactions might require more signatures from further regulated asset issuers.
+- The complicating factor of having approved, but not submitted transactions makes it difficult for approval servers to keep track of the state of the world.
+- One downside is that transactions involving multiple regulated assets are no longer possible.
```

Comment:
Why is this the case?

Reply:
This should actually say transactions involving multiple issuers, if there's two regulated assets by the same approval server its no problem.

If the transaction has 2 regulated assets it means it needs to be approved by 2 different approval servers.  If we have assets REGA and REGB, the REGA server wants to approve it but its not signed by REGB server, and vice versa.  In the original implementation we could get approved by REGA, get the tx signed by them, and then submit it to REGB server for signing.